### PR TITLE
Initial Pixhack V5 overview

### PR DESCRIPTION
**Pull request by peterbarker**
Initial Pixhack V5 overview fro provided docs.

### CODE REVIEW

File: common/source/docs/common-pixhackV5-overview.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+-  **Processor**
+
+   -  32-bit ARM Cortex M7 core with DPFPU
+   -  216 Mhz/512 KB RAM/2 MB Flash
+   -  32-bit co-processor
+
+-  **Sensors**
+
+   -  Dual accel/gyro IMUs (InvenSense ICM20689 and Bosch BMI055)
```
Comment (tridge) [inline common/source/docs/common-pixhackV5-overview.rst]: It has 3 IMUs, they are:

- ICM20689
- ICM20602
- BMI055

File: common/source/docs/common-pixhackV5-overview.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   -  USB Input: 4.75~5.25V
+   -  High-power servo rail, up to 36V
+      (servo rail does not power the flight controller)
+   -  Dual battery monitor inputs
+   -  Pixhack V5 can be triple redundant if power is provided
+      to both battery monitor inputs and the USB port
+
+-  **Interfaces**
+
+   -  6 IOMCU PWM servo outputs
```
Comment (tridge) [inline common/source/docs/common-pixhackV5-overview.rst]: it has 8 PWMs on IOMCU

File: common/source/docs/common-pixhackV5-overview.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+   -  8 FMU PWM servo outputs (D-Shot capable)
```
Comment (tridge) [inline common/source/docs/common-pixhackV5-overview.rst]: while there are 8 FMU PWMs, only 6 of them are exposed on pin headers

File: common/source/docs/common-pixhackV5-overview.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+   -  3 dedicated PWM/Capture inputs on FMU
+   -  S.Bus servo output
+   -  R/C inputs for CPPM, Spektrum / DSM and S.Bus
```
Comment (tridge) [inline common/source/docs/common-pixhackV5-overview.rst]: I don't believe it has a spektrum satellite port

File: common/source/docs/common-pixhackV5-overview.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+   -  Analogue / PWM RSSI input
+   -  5x general purpose serial ports
+   -  4x I2C ports
+   -  4x SPI buses
```
Comment (tridge) [inline common/source/docs/common-pixhackV5-overview.rst]: it has 5 SPI buses, but we only enable 3 at this time (to save on DMA channels)

**Comment by tridge** [comment]
might also be worth mentioning:

- the USB-C connector
- the builtin 3-colour LED in the cube
also worth checking if the retail version will come with the GPS, power brick, buzzer etc

**Comment by auturgy** [comment]
Also added to common-autopilots 

Regards,

James

**Pull request merged**